py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -342,3 +346,189 @@ def compile_module(
         settings.use_fast_partitioner = True
 
     return partitioned_module
+
+
+def convert_module_to_trt_engine(
+    module: torch.fx.GraphModule,
+    method_name: str = "forward",
+    inputs: Optional[Sequence[Input | torch.Tensor]] = None,
+    enabled_precisions: Optional[Set[torch.dtype | _enums.dtype]] = None,
+    debug: bool = DEBUG,
+    workspace_size: int = WORKSPACE_SIZE,
+    min_block_size: int = MIN_BLOCK_SIZE,
+    torch_executed_ops: Set[str] = field(default_factory=set),
```

Comment:
I am not sure if `field` is used outside of the context of an explicit `dataclass`, so this could alternatively just be `set()` or adapt from what is currently used in the other functions with this argument.